src/components/Room/Room.test.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Room from './Room';
+import useChatContext from '../../hooks/useChatContext/useChatContext';
+jest.mock('../../hooks/useChatContext/useChatContext');
+
+const mockUseChatContext = useChatContext as jest.Mock<any>;
+
+const mockToggleChatWindow = jest.fn();
+mockUseChatContext.mockImplementation(() => ({ setIsChatWindowOpen: mockToggleChatWindow }));
+
```

Comment:
I really like using snapshot tests - but I'm not sure if they are really needed here. I think that snapshot tests are great for testing lots of different things at once, but here we are testing only one thing. So I think that the `expect(wrapper.prop('className')).toContain('chatWindowOpen');` test that you already have is sufficient. Just need to add `expect(wrapper.prop('className')).not.toContain('chatWindowOpen');` to the first test.